x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_landing.scss

```
@@ -0,0 +1,13 @@
+.crawler-landing {
+  &__panel {
+    overflow: hidden;
+    background-image: url('./assets/bg_crawler_landing.png');
+    background-size: 45%;
+    background-repeat: no-repeat;
+    background-position: right -2rem;
+
+    > div {
+      max-width: 50rem;
+    }
+  }
+}
```

These are taken from the css for our crawler onboarding flow.  This is the css from our crawler onboarding in the standalone UI.  If there's a better more Kibana-y way to handle the background image, let me know!

---

Background image looks good to me! Small comment on the CSS though, we're generally preferring to switch to camel casing (so `.crawlerLanding` in this case). I'd also strongly suggest giving the `max-width` wrapper an actual className and not simply relying on a plain selector for it.

```suggestion
.crawlerLanding {
  &__panel {
    overflow: hidden;
    background-image: url('./assets/bg_crawler_landing.png');
    background-size: 45%;
    background-repeat: no-repeat;
    background-position: right -2rem;
  }

 &__wrapper {
    max-width: 50rem;
  }
}
```

---

I've addressed this in 0f3730b